### ISSUE TITLE
Gold slime mimics, made with grenades, take the form of the grenade

### ISSUE THREAD

**Issue by SonixApache** [d3athrow/vgstation13#20125]
#### Description of issue

Rather than taking on the form of something around them, these mimics show up as grenades. This limits their use significantly.
![image](https://user-images.githubusercontent.com/8468269/46878961-0a8edc00-ce45-11e8-846c-12f443d1305d.png)

#### Difference between expected and actual behavior
Expected: 
- Hold mimic grenade, spawn copies of myself
- Throw mimic grenade into a room, it makes copies of stuff in the room
- Put mimic grenade in chest, get mimic chests
- Basically just the handheld effects, but remotely

Actual:
![image](https://user-images.githubusercontent.com/8468269/46879037-314d1280-ce45-11e8-8d88-2068c7b9300a.png)

#### Steps to reproduce 
EX grenade, gold slime extract, beaker with water
Assemble & set off
#### Length of time in which bug has been known to occur
who knows, probably since implementation

#### Server revision
f774a274a2aed950cacbaac4e2a522c853c5389d

**Comment by Shadowmech88** [comment]
The way to fix this would be using `get_holder_at_turf_level()` instead of `loc`.
